frontend/src/components/landing-page/mock-top-cars.ts

```diff
@@ -0,0 +1,209 @@
+declare type TopCars = {
+  id: string;
+  name: string;
+  photoUrls: string[] | [];
+  brand: {
+    name: string;
+    logoUrl: string;
+  };
+  price: number;
+  complectationName?: string;
+  speedometr: number;
+  transmission: string;
+  location: string;
+  fuel: string;
+};
```

Comment:
@iryb 
Do we have the for such cars?

Reply:
@stanislav-ostr 
I remember you worked on the TopCar card. Do we have a type for the card's card?

Reply:
TopCarCard haven't any props. I created this  type TopCars localy just for correct rending TopCarCard element (using mock data). I can't image all properties which will require this element in future, this is why type TopCars is created here.

Reply:
Yes I didn't add any props to TopCarCard